### PR TITLE
they say start the year the way you want to spend it so let's REORG!

### PR DESCRIPTION
**Pull request by rehandalal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,6 +55,7 @@ <h1 id="answer" class="flex">WAIT</h1>
         <h2>Days since last reorg: <strong id="last-reorg-days"></strong></h2>
         <h2>Expected reorg <strong id="next-reorg-days"></strong></h2>
         <ul>
+            <li>(since January 19, 2020)</li>
```
Comment (mnoorenberghe) [inline index.html]: 2021?

Comment (rehandalal) [reply index.html]: D'OH!

**Review by bnjbvr — APPROVED** [review]
Uhh, approved, I guess? (we're "looking for maintainers", in case any of you is interested!)

**Comment by rehandalal** [comment]
I'm happy to hop on as a maintainer.

**Comment by adngdb** [comment]
Thanks Rehan! I've invited you, let me know if there's a problem.

**Pull request merged**